kormir-server/src/routes.rs

```diff
@@ -34,30 +34,82 @@ pub async fn list_events(
             "Failed to list events".to_string(),
         )
     })?;
+    match Format::from_query(&params) {
+        Err(_) => Err((StatusCode::BAD_REQUEST, "Invalid format".into())),
+        Ok(format) => match format {
```

Comment:
NIT: return an error here of Result<Value, Error> to properly handle panic in the format match